keyboards/keebio/levinson/keymaps/issmirnov/config.h

```diff
@@ -9,10 +13,22 @@
    // Enable animations. +5500 bytes
    #define RGBLIGHT_ANIMATIONS
 
+   // remove unused animations
```

Comment:
might be better to undef `RGBLIGHT_ANIMATIONS` and only explicitly define the ones you want.

Reply:
Done, thanks!